pkg/kubelet/kubelet.go

```diff
@@ -1521,6 +1538,10 @@ func (kl *Kubelet) tryUpdateNodeStatus() error {
 		return fmt.Errorf("no node instance returned for %q", kl.hostname)
 	}
 
+	if err := kl.reconcileCbr0(node.Spec.PodCIDR); err != nil {
```

Comment:
Seems odd to have this here. This function returns state to the master, the caller may not be aware of this mutating operation. Can we add to the function's doc?